### ISSUE TITLE
Bot initializes and stop aver evolution check. Logs do not report any errors.

### ISSUE THREAD

**Issue by baggednismo** [NECROBOTIO/NecroBot#5082]
## Stop! Before you create this issue (you can delete this section when opening the issue):
1. Have you validated that your config/auth.json and config/config.json is valid JSON? Use http://jsonlint.com/ to check.
Both JSON files have been validated

2. Have you searched to see if there are other issues for the same issue? If so, comment on that issue instead.
I was unable to locate the same issue already mentioned

### Expected Behavior
Bot to initialize and follow GPX path/snipe

### Actual Behavior
Bot starts and initializes then stops after evolution check.

This is the full extent of a clean log...
Initializing NecroBot logger at time 8/11/2016 9:13:45 AM...

### Your config.json
{
  "TranslationLanguageCode": "en",
  "AutoUpdate": false,
  "TransferConfigAndAuthOnUpdate": true,
  "UseWebsocket": true,
  "WebSocketPort": 14251,
  "StartupWelcomeDelay": false,
  "UseTelegramAPI": false,
  "TelegramAPIKey": null,
  "AmountOfPokemonToDisplayOnStart": 10,
  "DetailedCountsBeforeRecycling": false,
  "MaxBerriesToUsePerPokemon": 3,
  "CatchPokemon": false,
  "AutomaticallyLevelUpPokemon": false,
  "OnlyUpgradeFavorites": true,
  "UseLevelUpList": true,
  "AmountOfTimesToUpgradeLoop": 5,
  "GetMinStarDustForLevelUp": 5000,
  "LevelUpByCPorIv": "iv",
  "UpgradePokemonCpMinimum": 1000.0,
  "UpgradePokemonIvMinimum": 95.0,
  "UpgradePokemonMinimumStatsOperator": "and",
  "DisableHumanWalking": false,
  "DefaultLatitude": 42.448826434316871,
  "DefaultLongitude": -75.063931345939636,
  "WalkingSpeedInKilometerPerHour": 12.0,
  "MaxSpawnLocationOffset": 10,
  "FastSoftBanBypass": false,
  "DelayBetweenPlayerActions": 1,
  "DelayBetweenPokemonCatch": 1,
  "DumpPokemonStats": false,
  "EvolveAboveIvValue": 95.0,
  "EvolveAllPokemonAboveIv": false,
  "EvolveAllPokemonWithEnoughCandy": true,
  "EvolveKeptPokemonsAtStorageUsagePercentage": 90.0,
  "KeepPokemonsThatCanEvolve": false,
  "KeepMinCp": 1250,
  "KeepMinIvPercentage": 90.0,
  "KeepMinLvl": 6,
  "KeepMinOperator": "or",
  "UseKeepMinLvl": false,
  "PrioritizeIvOverCp": false,
  "KeepMinDuplicatePokemon": 0,
  "UseGpxPathing": true,
  "GpxFile": "My_Loop1.GPX",
  "VerboseRecycling": false,
  "RecycleInventoryAtUsagePercentage": 90.0,
  "RandomizeRecycle": false,
  "RandomRecycleValue": 5,
  "DelayBetweenRecycleActions": false,
  "UseEggIncubators": true,
  "minEggKmForLimitedIncubators": 2,
  "UseLuckyEggConstantly": false,
  "UseLuckyEggsMinPokemonAmount": 30,
  "UseLuckyEggsWhileEvolving": false,
  "UseIncenseConstantly": true,
  "UseBerriesMinCp": 1000,
  "UseBerriesMinIv": 90.0,
  "UseBerriesBelowCatchProbability": 0.2,
  "UseBerriesOperator": "or",
  "UseSnipeLocationServer": false,
  "SnipeLocationServer": "localhost",
  "SnipeLocationServerPort": 16969,
  "GetSniperInfoFromPokezz": true,
  "GetOnlyVerifiedSniperInfoFromPokezz": true,
  "GetSniperInfoFromPokeSnipers": true,
  "GetSniperInfoFromPokeWatchers": true,
  "SnipeWithSkiplagged": true,
  "MinPokeballsToSnipe": 100,
  "MinPokeballsWhileSnipe": 0,
  "MinDelayBetweenSnipes": 10000,
  "SnipingScanOffset": 0.003,
  "SnipeAtPokestops": true,
  "SnipeIgnoreUnknownIv": false,
  "UseTransferIvForSnipe": false,
  "SnipePokemonNotInPokedex": false,
  "RenamePokemon": false,
  "RenameOnlyAboveIv": true,
  "RenameTemplate": "{1}_{0}",
  "MaxPokeballsPerPokemon": 6,
  "MaxTravelDistanceInMeters": 1000,
  "TotalAmountOfPokeballsToKeep": 150,
  "TotalAmountOfPotionsToKeep": 20,
  "TotalAmountOfRevivesToKeep": 0,
  "TotalAmountOfBerriesToKeep": 200,
  "UseGreatBallAboveCp": 1000,
  "UseUltraBallAboveCp": 1250,
  "UseMasterBallAboveCp": 1500,
  "UseGreatBallAboveIv": 85.0,
  "UseUltraBallAboveIv": 95.0,
  "UseGreatBallBelowCatchProbability": 0.2,
  "UseUltraBallBelowCatchProbability": 0.1,
  "UseMasterBallBelowCatchProbability": 0.05,
  "EnableHumanizedThrows": false,
  "NiceThrowChance": 40,
  "GreatThrowChance": 30,
  "ExcellentThrowChance": 10,
  "CurveThrowChance": 90,
  "ForceGreatThrowOverIv": 90.0,
  "ForceExcellentThrowOverIv": 95.0,
  "ForceGreatThrowOverCp": 1000,
  "ForceExcellentThrowOverCp": 1500,
  "TransferWeakPokemon": false,
  "TransferDuplicatePokemon": true,
  "TransferDuplicatePokemonOnCapture": true,
  "FavoriteMinIvPercentage": 98.5,
  "AutoFavoritePokemon": true,
  "UsePokemonToNotCatchFilter": false,
  "UsePokemonSniperFilterOnly": false,
  "ItemRecycleFilter": [
    {
      "Key": "itemUnknown",
      "Value": 0
    },
    {
      "Key": "itemLuckyEgg",
      "Value": 200
    },
    {
      "Key": "itemIncenseOrdinary",
      "Value": 100
    },
    {
      "Key": "itemIncenseSpicy",
      "Value": 100
    },
    {
      "Key": "itemIncenseCool",
      "Value": 100
    },
    {
      "Key": "itemIncenseFloral",
      "Value": 100
    },
    {
      "Key": "itemTroyDisk",
      "Value": 100
    },
    {
      "Key": "itemXAttack",
      "Value": 100
    },
    {
      "Key": "itemXDefense",
      "Value": 100
    },
    {
      "Key": "itemXMiracle",
      "Value": 100
    },
    {
      "Key": "itemSpecialCamera",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasicUnlimited",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasic",
      "Value": 100
    },
    {
      "Key": "itemPokemonStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemItemStorageUpgrade",
      "Value": 100
    }
  ],
  "PokemonsNotToTransfer": [
    "gyarados",
    "ditto",
    "kabutops",
    "aerodactyl",
    "articuno",
    "zapdos",
    "moltres",
    "dragonite",
    "mewtwo",
    "mew"
  ],
  "PokemonsToEvolve": [
    "caterpie",
    "weedle",
    "pidgey",
    "rattata"
  ],
  "PokemonsToLevelUp": null,
  "PokemonsToIgnore": [
    "caterpie",
    "weedle",
    "pidgey",
    "rattata",
    "spearow",
    "zubat",
    "doduo"
  ],
  "PokemonsTransferFilter": {
    "Pikachu": {
      "KeepMinCp": 500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Golduck": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Farfetchd": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Krabby": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Kangaskhan": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 60.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Horsea": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Staryu": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "MrMime": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 40.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Scyther": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Jynx": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Electabuzz": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Magmar": {
      "KeepMinCp": 1400,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Pinsir": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Tauros": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Magikarp": {
      "KeepMinCp": 200,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Gyarados": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": true,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Lapras": {
      "KeepMinCp": 1800,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 80.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Eevee": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 95.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Vaporeon": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Jolteon": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Flareon": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Porygon": {
      "KeepMinCp": 1250,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Snorlax": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Blastoise": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Venusaur": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Charizard": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    },
    "Dragonite": {
      "KeepMinCp": 1500,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": true,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "or",
      "MovesOperator": "or"
    }
  },
  "PokemonToSnipe": {
    "Locations": [
      {
        "Latitude": 38.556807486461118,
        "Longitude": -121.2383794784546
      },
      {
        "Latitude": -33.859019,
        "Longitude": 151.213098
      },
      {
        "Latitude": 47.5014969,
        "Longitude": -122.0959568
      },
      {
        "Latitude": 51.5025343,
        "Longitude": -0.2055027
      }
    ],
    "Pokemon": [
      "magikarp",
      "venusaur",
      "charizard",
      "blastoise",
      "beedrill",
      "raichu",
      "sandslash",
      "nidoking",
      "nidoqueen",
      "clefable",
      "ninetales",
      "golbat",
      "vileplume",
      "golduck",
      "primeape",
      "arcanine",
      "poliwrath",
      "alakazam",
      "machamp",
      "golem",
      "rapidash",
      "slowbro",
      "farfetchd",
      "muk",
      "cloyster",
      "gengar",
      "exeggutor",
      "marowak",
      "hitmonchan",
      "lickitung",
      "rhydon",
      "chansey",
      "kangaskhan",
      "starmie",
      "mrMime",
      "scyther",
      "magmar",
      "electabuzz",
      "jynx",
      "gyarados",
      "lapras",
      "ditto",
      "vaporeon",
      "jolteon",
      "flareon",
      "porygon",
      "kabutops",
      "aerodactyl",
      "snorlax",
      "articuno",
      "zapdos",
      "moltres",
      "dragonite",
      "mewtwo",
      "mew",
      "ivysaur",
      "charmeleon",
      "wartortle",
      "charmander",
      "squirtle",
      "electrode",
      "omanyte",
      "omastar",
      "kabuto",
      "dratini",
      "dragonair"
    ]
  },
  "PokemonToUseMasterball": [
    "articuno",
    "zapdos",
    "moltres",
    "mew",
    "mewtwo"
  ]
}

### Steps to Reproduce
run the bot...

### Other Information
Version: 0.8.5
.NET Version: 4.6.2
Windows Version: 8.1 Pro x64
What is affected (catch, transfer, evolve, recycle, snipe, powerup/levelup, other): Bot basic functionality

**Comment by nfettinger** [comment]
Not reproducible.

**Comment by nfettinger** [comment]
We can not help you with setup or configuration items here. This space is for code development issues only. Please use our Discord chat room #support for further assistance.

**Comment by baggednismo** [comment]
nfettinger... this is not a config issue. this is your software not working. the bot crashed and did not dumped to the logs it is unable to run since that point. I have tried fresh release and fresh config... the bot does not function at all on this computer. I copied the entire app to another computer (that doesnt work) and it functions fine. there is something wrong with the app that it will not function on some machines. AGAIN, it was working for days and just all the sudden stopped.
